test/XRP/reliable-submission-xrp-client.test.ts

```diff
@@ -53,6 +43,17 @@ describe('Reliable Submission XRP Client', function (): void {
       fakedGetPaymentValue,
       fakedTransactionResultValue,
     )
+
+    this.fakedRawTransactionStatusValue = new RawTransactionStatus(
```

Comment:
could have also set `fakedRawTransactionStatusValue.validated = false` in here but the state changes are already interesting to reason about...felt better to just make a new one every time.